packages/components/src/internal/components/domainproperties/propertiesUtil.ts

```diff
@@ -185,6 +188,17 @@ export function removeUnusedOntologyProperties(obj) {
 }
 
 export function removeNonAppProperties(obj) {
+    delete obj.lookupContainer;
```

Comment:
I just moved these around so that the ordering of them is aligned with the order of property names within `reorderSummaryColumns`